examples/notebooks/eomt_semantic_segmentation.ipynb

```diff
@@ -0,0 +1,167 @@
+    "\n",
+    "> **Important**: When running on Google Colab make sure to select a GPU runtime for faster processing. You can do this by going to `Runtime` > `Change runtime type` and selecting a GPU hardware accelerator."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Installation\n",
+    "\n",
+    "LightlyTrain can be installed directly via `pip`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install lightly-train"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "> **Important**: LightlyTrain is officially supported on\n",
+    "> - Linux: CPU or CUDA\n",
+    "> - MacOS: CPU only\n",
+    "> - Windows (experimental): CPU or CUDA\n",
+    "> \n",
+    "> We are planning to support MPS for MacOS.\n",
+    "> \n",
+    "> Check the [installation instructions](https://docs.lightly.ai/train/stable/installation.html) for more details on installation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prediction using LightlyTrain's model weights\n",
+    "\n",
+    "### Download the model weights\n",
+    "\n",
+    "Here we download the ViT-S/16 weights trained on COCO-Stuff with the command below as an example:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -O lightlytrain_dinov3_eomt_vits16_cocostuff.pt https://lightly-train-checkpoints.s3.us-east-1.amazonaws.com/dinov3_eomt/lightlytrain_dinov3_eomt_vits16_cocostuff.pt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Download an example image\n",
+    "\n",
+    "After downloading the model weights, we download an example image for inference with the following command: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!wget -O cat.jpg https://upload.wikimedia.org/wikipedia/commons/3/3a/Cat03.jpg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Load the model weights to LightlyTrain\n",
+    "\n",
+    "Then, we load the model weights with LightlyTrain's `load_model_from_checkpoint` function and do inference on the example image with the `predict` method:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import lightly_train\n",
+    "\n",
+    "model = lightly_train.load_model_from_checkpoint(\n",
+    "    \"lightlytrain_dinov3_eomt_vits16_cocostuff.pt\",\n",
+    "    device=\"cpu\",  # or \"cuda\" if you have a GPU available\n",
```

Comment:
Don't set `device`. Then it will automatically load the model on GPU if a GPU is available.